OKR.md

```diff
@@ -1,49 +1,56 @@
-  - Provide brief explanations of how, why and where to use them.
-  - Document how to add new elemements. This project should expand to include reusable components and ui patterns as we create them.
-
-**[Brave](https://brave.com) is the first browser with "native" IPFS support:**
-
-- IPFS connectivity working out-of-the-box in Brave
-  - Embedded js-ipfs runs in Brave without installing external daemon
-  - User experience is the same as in Firefox when running external node
-- Support for native protocols in Location Bar and DOM elements (`href`, `src`)
-
-**Structure documentation and discussion around the primary Browser vendor concerns:**
-
-- Create specs for addressing on the Decentralized Web
-  - `dweb:` proposal is documented
-  - `ipfs://` (URL-based solution) is documented
-  - `Ways to deal/support the Content Origin Policy for IPFS links` are documented
-  - published at https://github.com/ipfs/specs/tree/master/dweb-addressing
-- Set up `arewedistributedyet.com` (domain TBC) as a communal call to action. ([in-web-browsers/#24](https://github.com/ipfs/in-web-browsers/issues/24))
-  - Publish the list of apis needed, with a tests where possible, a short summary on what it would allow, and a list of p2p projects that want it.    
+# IPFS in web browsers: Objectives and Key Results - Q1 2018
+
+**WORK IN PROGRESS. This document will change.**
+
+
+## We have a shared understanding of how IPFS _should work_ in web browsers
+
+_Focus on people. What are the user journeys; what are their goals. How will IPFS improve a web users day?_
+
+### Key results
+
+- Create **the** reference document for what users goals are when using IPFS as part of their daily web browsing.
+- Document user journeys that capture an ideal UX for achieving those goals.
+- Document how dat and beaker are being used for this today.
+- Start a regular distributed web meetup in London.
+
+
+## Browser developers understand the requirements of the decentralized web.
+
+_Support the conversation with browser vendors. Make it as easy as possible for them to see where we are at, what's needed and why._
+
+### Key results
+
+- **Publish `arewedistributedyet.com` as a communal call to action.**
+  - Collate the initial list of apis needed [in-web-browsers/#35](https://github.com/ipfs/in-web-browsers/issues/35)
+  - Publish the list, with a tests where possible, a short summary on what it would allow, and a list of p2p projects that want it. ([in-web-browsers/#24](https://github.com/ipfs/in-web-browsers/issues/24))
+- **Create a final draft spec for [addressing on the Decentralized Web](https://github.com/ipfs/specs/tree/master/dweb-addressing)**
+
+
+## Lower the barrier to entry for casual users
+
+_Improve the onboarding experience. Make it easy to use IPFS in the browser today._
+
+### Key results
+
+- **IPFS Desktop replaces go-ipfs as the recommended starting point for new users**
+  - **TBC!** _what are the plans for ipfs-desktop?_
```

Comment:
Have automatic releases with Jenkins you be a nice Key result but I think @VictorBjelkholm should be the one dictating if this should be a KR or not.

Reply:
Yes, it is a good KR.  To be specific:
- Jenkins job builds installer artifacts for all supported platforms from a specific [git tag](https://github.com/ipfs-shipyard/ipfs-desktop/tags)

I feel we should also add this KR:

- IPFS Desktop installers are signed  on MS Windows (Mac?) and do not cause any security warning on user's machine.

This is crucial for good onboarding experience.